experimental/components/MsGraph/Actions/GetContacts.cs

```diff
@@ -90,7 +90,10 @@ protected override async Task<List<CalendarSkillContactModel>> CallGraphServiceW
 
             if (people?.Count > 0)
             {
-                var existingResult = contactsResult.SelectMany(c => c.EmailAddresses).ToHashSet();
+                var result = contactsResult.SelectMany(c => c.EmailAddresses);
```

Comment:
I would just move the IEnumerable straight into the constructor.

Reply:
Sure!